fhir_models.gemspec

```diff
@@ -17,6 +17,10 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
+  spec.files.reject! { |file| file =~ /lib\/fhir_models\/examples|lib\/fhir_models\/definitions\/schema|lib\/fhir_models\/definitions\/valuesets/}
+  spec.files.reject! { |file| file =~ /lib\/fhir_models\/igs\/hl7.fhir.r4.core\/package_supplement\/expansions.json/}
+  spec.files.reject! { |file| file =~ /lib\/fhir_models\/igs\/hl7.fhir.r4b.core\/package_supplement\/expansions.json/}
+  spec.files.reject! { |file| file =~ /lib\/fhir_models\/igs\/hl7.fhir.r5.core\/package_supplement\/expansions.json/}
```

Comment:
Could we do this so that it isn't hard coded and won't break when we add a new version?

Reply:
Changed the regex to be version agnostic.

I've also changed the initial file gathering to be limited to those in `lib`, so none of the ones listed above are included anymore.

Reply:
I think that looks good, but we should still include `LICENSE`.

Reply:
Added back